.github/workflows/cd.yml

```diff
@@ -54,6 +49,12 @@ jobs:
           toolchain: ${{ matrix.rust }}
           args: --locked --release --target ${{ matrix.target }}
 
+      - id: install-secret-key
+        name: Install gpg secret key
+        run: |
+          cat <(echo -e "${{ secrets.GPG_SECRET }}") | gpg --batch --import
```

Comment:
Interesting syntax, why so complicated as opposed to `echo "${{ secrets.GPG_SECRET }}" | gpg --import` ?